tools/tdbg/tdbg_commands.go

```diff
@@ -474,14 +481,31 @@ func newAdminTaskQueueCommands(clientFactory ClientFactory) []*cli.Command {
 }
 
 func newAdminDLQCommands(clientFactory ClientFactory) []*cli.Command {
+	categoriesString := getCategoriesList()
 	commonFlags := []cli.Flag{
 		&cli.StringFlag{
-			Name:  FlagDLQType,
-			Usage: fmt.Sprintf("Type of DLQ to manage. (options: namespace, history)"),
+			Name: FlagDLQType,
+			Usage: fmt.Sprintf(
+				"Type of DLQ to manage. (options: namespace, history for v1; %s for v2)",
+				categoriesString,
+			),
 		},
 		&cli.StringFlag{
 			Name:  FlagCluster,
-			Usage: "Source cluster",
+			Usage: "Source cluster (v1 only)",
+		},
+		&cli.StringFlag{
+			Name:  FlagSourceCluster,
+			Usage: "Source cluster (v2 only)",
+		},
+		&cli.StringFlag{
+			Name:  FlagTargetCluster,
+			Usage: "Target cluster (v2 only)",
+		},
```

Comment:
shall we default to current cluster (target cluster of the tdbg command)?

Reply:
Modified to ask the AdminClient for the current cluster name and added tests for it.